### ISSUE TITLE
Getting IPN calls from demo

### ISSUE THREAD

**Issue by HenryMehta** [odero/django-pesapal#22]
I know this isn't an error with django-pesapal but I don't know where to ask this question.  I've got payments processing within demo to the point where they're pending, but they never go beyond pending.  How do I get the demo system to replicate actual paid amounts?

**Comment by odero** [comment]
Hey. Just seeing this. Did you find a solution?
pesapal used to have a mobilemoney test url which would generate a demo transaction code - but it no longer works. Now I think the way to do it would be to sign into demo.pesapal.com and: 
- load the sandbox/demo wallet with some test amount
- use the one of their test cards

**Comment by HenryMehta** [comment]
I never got it working and currently have it on hold.  I've tried using demo.pesapal.com but I cannot load any amount to the test account and Peaspal told me to use the live system to test it, which I won't do

**Comment by odero** [comment]
I have to admit the process for loading the eWallet is quite convoluted (I don't know why). You can follow the steps below.

1. If you've signed in to demo.pesapal.com you see the "Topup Wallet" on your dashboard.
2. When you click that it'll redirect you to a page where you can enter the amount of credits you'd like to buy.
3. Enter an amount like 5000. It'll then redirect you to a payment page. You'll see a field labeled "M-Pesa Confirmation Code:"
4. In a new tab go to https://demo.pesapal.com/dashboard/my/mobilemoneytest (you can find this link in the demo.pesapal landing page) and run a demo payment (See screenshot) and click "Generate test details". Copy the confirmation code you receive, into the "M-Pesa Confirmation Code" field in 3 above
![Screenshot 2019-07-16 at 10 48 28](https://user-images.githubusercontent.com/380686/61275798-5ea16500-a7b7-11e9-9933-6647efe37d69.png)
You might have to wait a while for it to the transaction to go through (again I don't know why). I suppose they might be queuing all demo requests and running them in batch. **_You could also ask them to load you eWallet on the demo account_**.

If it does go through, then next time you load the payment, you should use the "eWallet" payment option and the IPN should work as expected.

**Comment by HenryMehta** [comment]
Thanks.  I tried this initially but thought I would try again given your instruction and I've just found they did eventually process the transaction I had put through.  I've tried adding more to see if it comes through.

This now works when I process the payment during the intermediate screen.  But if I click Finish Ordering and then process the payment through my buyer PesaPal account, how should I get notification in the Django application?
